cmd/controller/main.go

```diff
@@ -50,6 +50,8 @@ var (
 	namespaceAll         = flag.Bool("all-namespaces", true, "Scan all namespaces or only the current namespace (default=true).")
 	additionalNamespaces = flag.String("additional-namespaces", "", "Comma-separated list of additional namespaces to be scanned.")
 	labelSelector        = flag.String("label-selector", "", "Label selector which can be used to filter sealed secrets.")
+	rateLimitBurst       = flag.Int("rate-limit-burst", 2, "Number of requests per burst for verify endpoint")
+	rateLimitDelay       = flag.Int("rate-limit-delay", 2, "Number of seconds to sleep between bursts for verify endpoint")
```

Comment:
Unless I'm missing something, I think this isn't really a delay. Looking at the [documentation](https://pkg.go.dev/github.com/throttled/throttled#RateQuota):

> MaxRate specified the maximum sustained rate of requests and must be greater than zero. MaxBurst defines the number of requests that will be allowed to exceed the rate in a single burst and must be greater than or equal to zero.

> Rate{PerSec(1), 0} would mean that after each request, no more requests will be permitted for that client for one second. Rate{PerSec(2), 0} permits one request per 0.5 seconds rather than two requests in one second. In practice, you probably want to set MaxBurst >0 to provide some flexibility to clients that only need to make a handful of requests. In fact a MaxBurst of zero will *never* permit a request with a quantity greater than one because it will immediately exceed the limit.

Reply:
I was biased by the http response that is talking about burst and delay, OK, got it, I will change that, thanks

Reply:
it's changed